packages/Widget/.babelrc

```diff
@@ -0,0 +1,46 @@
+{
+    "presets": [
+        ["@babel/preset-typescript"],
+        ["@babel/preset-env", {
+            "targets": {
+                "browsers": "defaults and supports webgl2"
+            },
+            "modules": false
+        }]
+    ],
+    "plugins": [
+        ["module-resolver", {
+            "cwd": "packagejson",
+            "root": ["./src"],
+            "extensions": [".js", ".ts", ".tsx"],
+            "alias": {
+                "@itowns/geodesy": "../Geodesy/src/Main.js",
+                "itowns": "../Main/src/Main.js"
+            }
```

Comment:
Once again this leads to incorrect compilation of path.

Reply:
removed